src/main/java/com/spotinst/sdkjava/model/SpotinstElastigroupClientGcp.java

```diff
@@ -17,8 +20,17 @@
     //region Members
     private              String                                    authToken;
     private              String                                    account;
+    private              ISpotinstElastigroupRepoGcp               spotinstElastigroupRepoGcp;
     //endregion
 
+    public ISpotinstElastigroupRepoGcp getSpotinstElastigroupRepoGcp() {
```

Comment:
If we are adding this common constructor to fetch repo object, make the change in all APIs in this class to use getSpotinstElastigroupRepoGcp() function to fetch repo object.

Reference - similar class for Azure.